commands/serverless.go

```diff
@@ -282,6 +341,41 @@ func RunServerlessConnect(c *CmdConfig) error {
 	return connectFromList(ctx, sls, list, c.Out)
 }
 
+// validateAccessKeyFormat validates that the access key follows the expected format
+func validateAccessKeyFormat(accessKey string) error {
```

Comment:
Since, access-key will be sent by out backend, do we really need to validate its format?

Reply:
I see this is being used in `connect` command. My initial thought was that we needed a stronger check to differentiate between legacy and new access keys. We simply don't want users to pass legacy access keys to the new connect command/flag and then suddenly break their environments due to deprecation. If this prevents that, let's keep it.